axon-framework/monitoring/tracing.md

```diff
@@ -0,0 +1,535 @@
+
+
+### Queries
+Queries support tracing in all of their forms. In order to be clear about how they work, 
+this section is split based upon the query's type. 
+For all types, the created spans will differ based on whether Axon Server is used or not. 
+The spans that are only available with Axon Server are marked as such.
+
+#### Direct queries
+Direct queries fetch a single result (either a single item or a single list) and receive no updates. 
+Their traces will look like the following table:
+
+| Trace name                                                | Description                                                                           |
+|-----------------------------------------------------------|---------------------------------------------------------------------------------------|
+| `AxonServerQueryBus.query(${queryName})`                  | (Only with Axon Server) The requesting service is dispatching the query.              |
+| `QueryProcessingTask(${queryName})`                       | (Only with Axon Server) The handling service is handling the query request in a task. |
+| `SimpleQueryBus.query(${queryName})`                      | The handling service is handling the query.                                           |
+| `AxonServerQueryBus.ResponseProcessingTask(${queryName})` | (Only with Axon Server) The requesting service is processing the response.            |
+
+#### Streaming queries
+Streaming queries, introduced in Axon Framework 4.6.0, look very similar to the traces of a Direct query. 
+They do not contain a `ResponseProcessingTask` span since their results are directly published to the invoker of the query.
+Their traces will look like the following table:
+
+| Trace name                                        | Description                                                                           |
+|---------------------------------------------------|---------------------------------------------------------------------------------------|
+| `AxonServerQueryBus.streamingQuery(${queryName})` | (Only with Axon Server) The requesting service is dispatching the query.              |
+| `QueryProcessingTask(${queryName})`               | (Only with Axon Server) The handling service is handling the query request in a task. |
+| `SimpleQueryBus.query(${queryName})`              | The handling service is handling the query.                                           |
+
+#### Scatter-Gather queries
+Scatter-Gather queries are like a direct query but can fetch results from multiple services at the same time. 
+Part of the trace can thus be duplicated multiple times, since multiple services are invoked.
+Their traces will look like the following table:
+
+| Trace name                                             | Description                                                                                          |
```

Comment:
I'd have a similar code-block suggestion for the AxonServer and non-AxonServer samples, just as with the Commands section.